Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,432 @@
+  | extend RequestURL_ = iif(RequestURL_ matches regex @"^(.{2,6}=.{1,6})", extract(@"(?:\d;|.{1}\w.{1};)(?P<url>[a-zA-Z0-9_*.,}{&%$~:;\-=\/?[:space:]]+)", 1, RequestURL_), iif(RequestURL_ matches regex @"^\w=\d$", "", RequestURL_))
+  | lookup Actions on fw_action
+  // Sets the mandatory EventResult based on the DvcAction.
+  | extend EventResult = case(DvcAction == "Allow", "Success",
+                            DvcAction == "Management", "NA",
+                            DvcAction == "NA", "NA",
+                            "Failure"
+                            )
+  | extend LogMsgSeverity = case(LogSeverity == 10, "Emergency (0)",
+                                              LogSeverity == 9, "Alert (1)",
+                                              LogSeverity == 8, "Critical (2)",
+                                              LogSeverity == 7, "Error (3)",
+                                              LogSeverity == 6, "Warning (4)",
+                                              LogSeverity == 5, "Notice (5)",
+                                              LogSeverity == 4, "Info (6)/Debug (7)",
+                                              LogSeverity == 3, "Not Mapped (3)",
+                                              LogSeverity == 2, "Not Mapped (2)",
+                                              LogSeverity == 1, "Not Mapped (1)",
+                                              "Not Mapped"
+                                              )
+  | extend EventSeverity = case(tolong(LogSeverity) <= 4, "Informational",
+                                              tolong(LogSeverity) <= 6, "Low",
+                                              tolong(LogSeverity) <= 8, "Medium",
+                                              tolong(LogSeverity) > 8, "High",
+                                              ""
+                                              )
+  | extend HttpRequestMethod = case(RequestMethod == 0, "No Operation (0)",
+                                              RequestMethod == 1, "HTTP GET (1)",
+                                              RequestMethod == 2, "HTTP POST (2)",
+                                              RequestMethod == 3, "HTTP HEAD (3)",
+                                              ""
+                                              )
+  | extend NetworkProtocolVersion = case(DestinationIP has ".", "IPv4"
+                                            , DestinationIP has ":", "IPv6"
+                                            , ""
+                                            )
+    , NetworkProtocol = toupper(trim_end(@"/.*", Protocol))
+  | project-rename
+    EventResourceId = _ResourceId
+    , DstMacAddr = DestinationMACAddress
+    , SrcMacAddr = SourceMACAddress
+    , DeviceModel = DeviceProduct
+    , FirmwareVersion = DeviceVersion

```

Comment:
not a network session field. can we map with [EventProductVersion](https://learn.microsoft.com/en-us/azure/sentinel/normalization-common-fields#eventproductversion). Is that a good fit?